### PR TITLE
Delete adjacent file concept and improve file match output

### PR DESCRIPTION
**Pull request by viktigpetterr**
- [x] Delete adjacent file concept
- [x] Improve file match output
- [x] Default snippet analysis to false to improve scan time.
- [x] Update tests to cover new output
- [x] Update picture in README.md
- [ ] Update [debricked-scan](https://bitbucket.org/debricked/debricked-scan) after release. Especially  `--disable-snippets` must be adjusted to the new flag.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -89,6 +89,12 @@
         ],
         "post-update-cmd": [
             "@auto-scripts"
+        ],
+        "cs": "vendor/bin/php-cs-fixer fix --config=.php-cs-fixer.dist.php -v --dry-run --stop-on-violation --diff --using-cache=no",
```
Comment (sweoggy) [inline composer.json:93]: I know this is from before, but I think we should use cache, so that we can speed up our GitHub Action a bit

Comment (viktigpetterr) [reply composer.json]: 371d48ca7789b95ebccf2150259c36406b19f6b3 @sweoggy

**Review by zozs — APPROVED** [review]
Looks good!

[debricked-scan](https://bitbucket.org/debricked/debricked-scan) needs to be updated after this has been merged as well, since the flag for `--disable-snippets` has been changed.

**Comment by viktigpetterr** [comment]
> Looks good!
> 
> [debricked-scan](https://bitbucket.org/debricked/debricked-scan) needs to be updated after this has been merged as well, since the flag for `--disable-snippets` has been changed.

Good that you said it, I will put it as a task.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 [![Build Status](https://github.com/debricked/debricked-cli/actions/workflows/test.yml/badge.svg?branch=master)](https://github.com/debricked/debricked-cli/actions/workflows/test.yml)
 [![Latest Stable Version](https://poser.pugx.org/debricked/cli/v/stable)](https://packagist.org/packages/debricked/cli)
 
-![Debricked CLI in action](debricked-cli.png)
+![Debricked CLI in action](debricked-spaceship.svg)
```
Comment (sweoggy) [inline README.md:5]: The spaceship is sexy, but I can kind of liked that we showed off the CLI (image could do with an update though and perhaps but somewhere else in the README)

Comment (sweoggy) [reply README.md:5]: If kept like this, the ALT text needs to be changed

Comment (viktigpetterr) [reply README.md:5]: I will create a updated one then

Comment (viktigpetterr) [reply README.md]: ce3e9806def26df95389ab8dca0abaaa36b543ef @sweoggy

Comment (viktigpetterr) [reply README.md]: deaffe87a3032a60de051371b3917f4ce7eae5cb @sweoggy

File: src/Model/FileGroup.php [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    }
+
+    public function addLockFile(SplFileInfo $lockFile): void
+    {
+        $this->lockFiles[] = $lockFile;
+    }
+
+    /**
+     * Returns `true` if the dependency file has a lock file. Otherwise `false`.
+     * If the FileGroup is a lockFileGroup then `true` is always returned.
```
Comment (viktigpetterr) [inline src/Model/FileGroup.php:55]:
```suggestion

```

**Comment by sweoggy** [comment]
Thanks!

**Pull request merged**